README.md

```diff
@@ -61,21 +65,17 @@ See [the makefile](https://github.com/guardian/guui/blob/master/makefile) for th
 ## IDE setup
 
 ### VSCode
-We use [babel-plugin-module-resolver](https://github.com/tleunen/babel-plugin-module-resolver) to allow us to import things cleanly (You need to have Xcode installed as the plugin installer uses `xcode-select`). If you're using VSCode, adding the following to a `./jsconfig.json` file will help it resolve modules usefully:
 
-```json
-{
-    "compilerOptions": {
-        "baseUrl": "src/",
-    }
-}
-```
+We recommend using the Visual Studio Code (VSCode) IDE. VSCode includes built in TypeScript language support and has a rich library of extensions which let you add formatters, debuggers,and tools to your installation to support your development workflow.
 
-The following extensions will life easier:
+You can download VSCode from [https://code.visualstudio.com/](https://code.visualstudio.com/)
 
-- [tslint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint) (esp. with `tslint.autoFixOnSave` enabled)
+Recommended VSCode extensions are listed in `.vscode/extensions.json` and VSCode should prompt you to install these when you open the project. You can also find these extensions Command Palette (shift + cmd + P) and typing the 'Extensions: Show Recommended Extensions command' or by searching for '@recommended' in the extensions pane.
 
-Also:
+#### Auto fix on save
 
-- [css-in-js syntax highlighting](https://marketplace.visualstudio.com/items?itemName=jpoissonnier.vscode-styled-components)
-- [colour highlighting in JS](https://marketplace.visualstudio.com/items?itemName=naumovs.color-highlight)
+We recommend you update your workspace setting to auto fix prettier errors on save. You can use the Command Palette (shift + cmd + P) to open user or workspace settings, just type in `Preferences: Open Workspace Settings`. You can then search for 'tslint.autoFixOnSave'. In your `settings.json` then set:
```

Comment:
Maybe something like:

We recommend you update your workspace settings to automatically fix formatting errors on save. 

1. Open the Command Palette (shift + cmd + P) and type `Preferences: Open Workspace Settings`
2. Search for `tslint.autoFixOnSave`
3. Update your `settings.json`:

Reply:
Nice suggestions 👍